test_rclcpp/test/test_timer.cpp

```diff
@@ -59,7 +59,7 @@ TEST(CLASSNAME(test_time, RMW_IMPLEMENTATION), timer_fire_regularly) {
     printf("spin_node_some() for 4s\n");
     while (std::chrono::steady_clock::now() < start + 4.5 * period) {
       executor.spin_node_some(node);
-      std::this_thread::sleep_for(period / 25);
```

Comment:
Not really.

Before it was 1000 / 25 = 40ms.
Afterwards it is 100 / 10 = 10ms.

And usually sleeping less then 10ms doesn't make much sense anyway.